internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -490,6 +490,16 @@ func createWindowsContainerDocument(ctx context.Context, coi *createOptionsInter
 	v2Container.RegistryChanges = &hcsschema.RegistryChanges{
 		AddValues: registryAdd,
 	}
+
+	if oci.IsIsolatedJobContainer(coi.Spec) {
+		v2Container.ContainerType = "HostProcess"
```

Comment:
This isnt a valid ContainerType in the product code. I dont think we should reuse it like this unless we get a host side change that this becomes a valid value

Reply:
`ContainerType` is being added to the HCS V2 schema as part of the effort to support HostProcess containers.
This PR builds on top of the other changes on HCS/GCS side.

Reply:
Sent you the PR offline.